SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java

```diff
@@ -61,6 +61,8 @@
 
     void onLowMemory();
 
+    void onNewIntent(Intent intent);
```

Comment:
If there is only a google implementation of this then perhaps only add it to the google ISwrve subclass?